web/src/components/TimeSlider.tsx

```diff
@@ -3,16 +3,14 @@ import { scaleLinear } from 'd3-scale';
 import { useNightTimes } from 'hooks/nightTimes';
 import { useDarkMode } from 'hooks/theme';
 import { useAtom } from 'jotai';
+import { ReactElement } from 'react';
+import { FaArrowsLeftRight, FaMoon, FaSun } from 'react-icons/fa6';
```

Comment:
I think the sun icon looks pretty bad, especially on a non-retina screen:

![Screenshot 2024-07-16 at 20 46 25](https://github.com/user-attachments/assets/753e5e89-d853-43ae-a434-cbc277ad8eee)

I'd much rather use the existing icons, but turn them into inline React components instead as a first step 🤷